### DISCUSSION TITLE
Silence the ble:exit message?

### DISCUSSION THREAD

**Discussion by thewithz** [akinomyoga/ble.sh#113] [Q&A] [answered]
I was looking through the wiki trying to find a way to silence the ble.sh error output `[ble: exit 127]` but wasn't able to find it. How can i achieve this?

**Comment by akinomyoga** [comment, marked as answer]
Look at [here](https://github.com/akinomyoga/ble.sh/wiki/Q&A#q10-how-to-turn-off-the-exit-status-mark-ble-exit-)

**Reply by akinomyoga** [reply]
Also, [§4.13.11](https://github.com/akinomyoga/ble.sh/wiki/Manual-%C2%A74-Editing#user-content-bleopt-exec_errexit_mark) is the corresponding item in the manual.
